Scripts/"?|.sh

```
@@ -0,0 +1,456 @@
+        echo -e "${RED}❌ $failed_links DocC link(s) failed validation${NC}"
+    else
+        echo -e "${GREEN}✅ All DocC internal links are valid${NC}"
+    fi
+}
+
+# Function to validate Swift symbol references
+validate_swift_symbols() {
+    echo -e "\n${BLUE}🔧 Validating Swift Symbol References...${NC}"
+    
+    # Find all double-backtick symbol references in SyntaxKit docs
+    local symbol_refs=$(grep -h -o '``[^`]*``' Sources/SyntaxKit/Documentation.docc/**/*.md 2>/dev/null | sort -u || true)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)